web/src/cypress/integration/schedules.ts

```diff
@@ -215,6 +225,34 @@ function testSchedules(screen: ScreenFormat) {
       cy.get('body').should('not.contain', rot.name)
     })
 
+    it('should create multiple rules on an assignment', () => {
+      cy.pageFab('Rotation')
+
+      cy.get('input[name=Sunday]').click()
+      cy.get('button[aria-label="Add rule"]').click()
+
+      cy.get('td')
```

Comment:
Is there a better/more specific selector we can use? `td` is a bit generic, also this is page-wide and not scoped to the dialog.